x/protorev/keeper/msg_server.go

```diff
@@ -149,9 +149,6 @@ func (m MsgServer) SetBaseDenoms(c context.Context, msg *types.MsgSetBaseDenoms)
 		m.k.DeleteAllPoolsForBaseDenom(ctx, baseDenom.Denom)
 	}
 
-	// Delete the old base denoms
-	m.k.DeleteBaseDenoms(ctx)
```

Comment:
We no longer need to delete the base denoms since we overwrite the single BaseDenoms value.